helm-charts/splunk-otel-collector/values.yaml

```diff
@@ -329,6 +329,12 @@ logsCollection:
     excludePaths: []
     # Boolean for ingesting the agent's own log
     excludeAgentLogs: true
+    # Field name conventions to use. (Only for those who are migrating from Splunk Connect for Kubernetes helm chart)
+    fieldNameConvention:
+      # Boolean for renaming pod metadata fields to match to Splunk Connect for Kubernetes helm chart.
+      renameFieldsSck: false
+      # Boolean for keeping Otel convention fields after renaming it
+      keepOtelContention: false
```

Comment:
This is applicable to metrics as well, right? I think it should be under `splunkPlatform` config group?

And do we really need an option to send both attributes? Maybe `nameConvention: <otel|sck>`?

Reply:
okay I will move to splunkPlatform. 
as for the use case for both, maybe for during migration phase, we don't want to break their already set up Splunk queries and stuff, but also they want to correlate logs with data in O11y.

Reply:
Ok, sounds good

Reply:
Do you still plan to move it under `splunkPlatform` and apply the same renaming to metrics? Or it's needed for logs only?

Reply:
There is also a typo: conVention

Reply:
And please make it `true` by default, otherwise all attributes removed